ca_description/urdf/sensors/cliff_sensors_gazebo.xacro

```diff
@@ -7,6 +7,7 @@
   <xacro:property name="max_range_m" value="0.04"/>
 
   <xacro:property name="rate_hz" value="20.0"/>
+  <xacro:property name="min_cliff_value" value="0.035"/>
 
   <gazebo reference="${name_side_left}_cliff_sensor_link">
```

Comment:
One improvement here:

- The block "sensor + gazebo plugin" can be simplified creating a macro and using it here 4 times.